icaruscode/NuGraphIcarus/scripts/setup_tritonserver-nugraph-v0_grid.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+unset PYTHONHOME
+unset PYTHONPATH
+rm -f tritonserver_nugraph-v0.log
+export APPTAINER_BIND=/etc/hosts,/tmp,/cvmfs
+#
+export BASEPORT=8000
+export NPORTS=3
+#
+export HTTPPORT=$BASEPORT
+export GRPCPORT=$((BASEPORT+1))
+export METRPORT=$((BASEPORT+2))
+#
+if [ -z "$FCL" ]; then 
+    export FCL=$1
+fi
+#
+while 2>/dev/null >"/dev/tcp/0.0.0.0/$BASEPORT"; do
+    BASEPORT=$((BASEPORT+NPORTS))
+    export HTTPPORT=$BASEPORT
+    export GRPCPORT=$((BASEPORT+1))
+    export METRPORT=$((BASEPORT+2))
+done
+#
+echo "physics.producers.NuGraphCryoE.TritonConfig.serverURL: \"localhost:"$GRPCPORT"\"" >> $FCL
```

Comment:
I _suggest_ to create a copy of the FHiCL file instead of changing the one specified, to prevent errors where the original FHiCL file is read-only or in storage that does not support appending.